packages/survey/src/Question/Item/Item.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  Box,
+  UseCheckboxGroupReturn,
+  UseRadioGroupReturn,
+  useCheckbox,
+  useRadio,
+} from '@chakra-ui/react'
+
+import { Checkbox } from '../Item/Checkbox'
+import { Radio } from '../Item/Radio'
+import { ItemGroupProps } from '../ItemGroup'
+import { itemStyle } from './style'
+
+type ItemProps = {
+  index?: number
+  label: string
+  isChecked: boolean
+} & Pick<ItemGroupProps, 'activeColor' | 'type' | 'withLabels'> &
```

Comment:
You should not import props from the calling file. Always from "outside" to "inside". This is dirty code not dos not follow the Dependency Inversion Principle. It's one of the [SOLID principles](https://en.wikipedia.org/wiki/SOLID).
So here you declare the props in this file and gather them like via `Pick<ItemProps, 'activeColor' | 'type' | 'withLabels'>` in the `ItemGroup` file.

You can read more about it at [clean-code-typescript from labs42io](https://github.com/labs42io/clean-code-typescript#dependency-inversion-principle-dip) or in my german translation at [mheob/clean-code-typescript](https://github.com/mheob/clean-code-typescript#prinzip-der-inversion-von-abh%C3%A4ngigkeiten---dependency-inversion-principle-dip).